app/controllers/vendors_controller.rb

```diff
@@ -71,6 +73,18 @@ def destroy
     end
   end
 
+  def favorite
+    user_id = current_user.id
+    deleted_value = @vendor.favorite_user_ids.delete(user_id)
+    @vendor.favorite_user_ids.push(user_id) if deleted_value.nil?
+    @vendor.save!
+    @vendors = Vendor.accessible_by(current_user).order(:updated_at => :desc)
+    @vendors = @vendors.sort_by { |a| (a.favorite_user_ids.include? current_user.id) ? 0 : 1 }
+    respond_with(@vendors.to_a) do |f|
+      f.js {}
```

Comment:
Is this `f.js {}` in the `do` block necessary here? It's in this `favorite` method, but not the equivalent in `products_controller.rb`